lib/rmt/cli/products.rb

```diff
@@ -16,7 +16,7 @@ def list
 
     if products.empty?
       if options.all
-        warn _('Run `%{command}` to synchronize with your SUSE Customer Center data first.') % { command: 'rmt-cli sync' }
+        warn _('Run \'%{command}\' to synchronize with your SUSE Customer Center data first.') % { command: 'rmt-cli sync' }
```

Comment:
Wouldn't it be easier to do it like `warn _("Run 'foo' to bar.")` so you don't have to escape single quotes inside single quotes?
Same for some other places.